pertpy/tools/_metadata.py

```diff
@@ -0,0 +1,71 @@
+import anndata
+import pandas as pd
+
+
+class MetaData:
+    """Python implementation of MetaData."""
+
+    def __init__(self):
+        self.cell_line_meta = pd.read_csv("https://ndownloader.figshare.com/files/35020903")
+
+    def annotate_cell_lines(
+        self,
+        adata: anndata,
+        cell_line_column: str = "DepMap_ID",
+        cell_line_type: str = "DepMap_ID",
+        cell_line_information: list = None,
+    ):
+        """Fetch cell line annotation.
+
+        For each cell, we fetch cell line annotation from Dependency Map (DepMap).
+
+        Args:
+            adata: The data object to annotate.
+            cell_line_column: The column of `.obs` with cell line information.
```

Comment:
Could you please add the defaults here? Like this
```suggestion
            cell_line_column: The column of `.obs` with cell line information. (default: "DepMap_ID")
```